app/Http/Controllers/Portal/Contact/ContactController.php

```diff
@@ -483,6 +485,9 @@ protected function updateAddress($contact, $addressData, $addressType, $projectI
                 }else{
                     $address->fill($this->arrayKeysToSnakeCase($addressData));
 
+                    $addressController = new AddressController();
+                    $checkAddressOk = $addressController->checkAddress($contact, $address, $projectId, true);
```

Comment:
Functionaliteit in Trait plaatsen zoals eerder vermeldt. Een nieuwe controller instance aanmaken lijkt mij niet de bedoeling.